tests/e2e/tests/subscription.spec.js

```diff
@@ -0,0 +1,204 @@
+const { test, expect } = require('@playwright/test');
+
+test.describe('Weather Subscription', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
```

Comment:
This is not enough when your web is heavy. Consider adding any validation that ensures your cite is loaded (e.g. critical ui elements are visible)